yang-semver/draft-ietf-netmod-yang-semver.txt

```diff
@@ -526,7 +526,8 @@ Internet-Draft                 YANG Semver                 November 2021
             to "X.Y.Z+1_compatible".
 
        iii  "X.Y.Z_non_compatible" - the artifact version SHOULD be
-            updated to "X.Y.Z+1_non_compatible".
+            updated to "X.Y.Z+1_non_compatible" (to ensure that
```

Comment:
I don't think that we need these two clarifications.  I think that the previous additional paragraph that you added is sufficient, and this section should just be the update rules.

Reply:
OK. I was debating that as well. Removed those.